app/controllers/hmis_api/projects_controller.rb

```diff
@@ -0,0 +1,45 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# TEMPORARY: example of a POST request for testing HMIS authentication. Replace with GraphQL endpoint!
+
+class HmisApi::ProjectsController < HmisApi::BaseController
+  include ArelHelper
+
+  def index
+    render json: project_scope.pluck(:ProjectName)
+  end
+
+  private def project_types
+    return HUD.project_types.keys unless project_params[:project_types].present?
+
+    @project_types ||= begin
+      types = []
+
+      project_type_to_id = project_source::PERFORMANCE_REPORTING.merge(project_source::RESIDENTIAL_PROJECT_TYPES)
```

Comment:
What is this line trying to do?  It looks like the merge adds:
`:permanent_housing=>[3, 9, 10, 13], :transitional_housing=>[2], :emergency_shelter=>[1], :street_outreach=>[4], :safe_haven=>[8]`

Reply:
In any case, this is dummy code that should be deleted soon!

Reply:
At least I'm consistent in my reviews!  I still think that should move to Project eventually.  Since this is just a test/example, let's throw a "not production" line in there somewhere so we can't accidentally run it in production.

Reply:
@awisema is going to delete this controller on his branch where he is adding the graphql endpoint.

I already have a comment about that at the top of this file, and the route is hidden behind `ENABLE_HMIS_API` endpoint, so it wont be exposed in production.

Reply:
I can change the route check to `if ENV['ENABLE_HMIS_API'] == 'true' && !Rails.env.production?`, is that sufficient?

Reply:
If we need an ENV to enable the whole thing, that's plenty.  Thanks!